README.md

```diff
@@ -17,7 +19,7 @@ import {
 
 ### setConfig
 
-The constructor takes one parameter "values" which an object which contains configuration for the following environments: `local, development, staging, production`;
+The constructor takes one parameter, "values", which is an object which contains configuration for the following environments: `local, development, staging, production`;
```

Comment:
@srutherfordbcg maybe we just move the environment "list" from the bottom up here. Because you can't really use the config if you haven't set one up. and maybe add a line that says that explicitly.

Reply:
Sounds like a good idea to me, I will move it